src/components/header.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+
+import LoginButton from "./login-button";
+import LogoutButton from "./logout-button";
+
+export default async function Header() {
+  return (
+    <header className="flex h-16 w-full items-center justify-between border border-blue-500 bg-cooper-blue-400 p-4">
+      <h1 className="text-3xl font-extrabold text-white">COOPER</h1>
```

Comment:
Another small change, can you make this a hyperlink that redirects the user back to the home (`/`) route? This will improve UX since users usually click on the logo / website name to go back to the homepage!